scripts/us_epa/parent_company/download_existing_facilities.py

```diff
@@ -0,0 +1,34 @@
+"""A simple script to download existing Facilities in Data Commons."""
```

Comment:
please add copyright header to this and the other new files

Reply:
Done